frontend/src/components/login.js

```diff
@@ -7,8 +7,10 @@ const Login = ({ setUserState }) => {
     const [errorState, setErrorState] = useState('');
 
     const handleLogin = event => {
+        //trim username and set to lowercase
```

Comment:
We should make the password & user input trim consistent

Reply:
addressed